plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -587,6 +589,62 @@ protected void verifyColumnNameLengthFailurePermissible(Throwable e)
         assertThat(e).hasMessageMatching("Column name must be shorter than or equal to '128' characters but got '129': '.*'");
     }
 
+    @Test
```

Comment:
These tests look very similar to the ones in MySQL connector. Is it reasonable to put them in jdbc module as an abstract class?

Reply:
Our initial implementation was to add in `BaseJdbcConnectorTest` - but not all JDBC based source supports it directly (Only MySQL and SqlServer returns a `ResultSet` - some of them works only with output parameter and some returns `ResultSet` with no fields. Plus both MySQL and SQLServer has a different way of creating and executing these stored procedures - and it might be some sort of overkill to merges these tests now.

Reply:
If you believe this is too much of a hassle, I am fine with leaving that as it is.